js/build.js

```diff
@@ -3,27 +3,29 @@ Fliplet.Widget.instance('image-gallery', function (data) {
   var photoswipeTemplate = Fliplet.Widget.Templates['templates.photoswipe'];
   var wallSelector = '[data-image-gallery-id=' + data.id + '] .wall:not("[data-mce-bogus] [data-image-gallery-id=' + data.id + '] .wall")';
 
-  function initGallery() {
+  function initGallery(options) {
     var $wall = $(wallSelector);
     var $bricks = $();
 
-    // Update remote image URLs to authenticated URLs
-    _.forEach(data.images, function (image, index) {
-      var $img = $('<img />');
+    if (options.appendImages) {
+      // Update remote image URLs to authenticated URLs
+      _.forEach(data.images, function (image, index) {
+        var $img = $('<img />');
 
-      image.url = Fliplet.Media.authenticate(image.url);
-      $img.on('load', function() {
-        reloadWall();
-      });
-      $img.attr('src', image.url);
-      $img.attr('alt', image.title);
+        image.url = Fliplet.Media.authenticate(image.url);
+        $img.on('load', function() {
+          reloadWall();
+        });
+        $img.attr('src', image.url);
+        $img.attr('alt', image.title);
 
-      var $brick = $('<div class="brick"></div>');
+        var $brick = $('<div class="brick"></div>');
 
-      $brick.append($img);
-      $bricks = $bricks.add($brick);
-    });
-    $wall.append($bricks);
+        $brick.append($img);
+        $bricks = $bricks.add($brick);
+      });
+        $wall.append($bricks);
```

Comment:
@upplabs-alex-levchenko @YaroslavOvdii Here's an error that should not be picked up by us.

Bad indentation.

```suggestion
      $wall.append($bricks);
```

Reply:
Yeah. My bad. Fixed